web_src/js/publicpath.js

```diff
@@ -1,6 +1,6 @@
 // This sets up the URL prefix used in webpack's chunk loading.
 // This file must be imported before any lazy-loading is being attempted.
 import {joinPaths} from './utils.js';
-const {AssetUrlPrefix} = window.config;
+const {assetUrlPrefix} = window.config;
 
-__webpack_public_path__ = joinPaths(AssetUrlPrefix, '/');
+__webpack_public_path__ = joinPaths(assetUrlPrefix, '/');
```

Comment:
```suggestion
__webpack_public_path__ = joinPaths(window.config.assetUrlPrefix, '/');
```
Since GitHub suddenly decided to break multiline suggestions, here you have it in two separate suggestions.

Reply:
No, I won't touch existing logic if it is unrelated during the refactoring. I only change the code it has to be changed.